ballerina/client.bal

```diff
@@ -81,11 +81,19 @@ public isolated client class Client {
         'class: "io.ballerina.lib.milvus.Client"
     } external;
 
-    # Searches for data in a collection in the Milvus vector database.
-    # 
+    # Conducts a vector similarity search for data in a collection in the Milvus vector database.
+    #
     # + request - The request to search for data
     # + return - If the search is successful, the result will be an array of arrays of `SearchResult` records. Unless, returns an error
     remote isolated function search(SearchRequest request) returns SearchResult[][]|Error = @java:Method {
         'class: "io.ballerina.lib.milvus.Client"
     } external;
+
+    # Conducts a scalar filtering for data in a collection in the Milvus vector database.
+    #
+    # + request - The request to search for data
+    # + return - If the search is successful, the result will be an array of arrays of `SearchResult` records. Unless, returns an error
```

Comment:
```suggestion
    # + return - On success, returns a two-dimensional array of `SearchResult` records. Otherwise, returns an `Error`.
```

Reply:
Fixed in [9592a2f](https://github.com/ballerina-platform/module-ballerinax-milvus/pull/7/commits/9592a2f4b36aa23d2888c9f135982ca78bdb1b77)